cmd/kubeops/internal/handler/handler.go

```diff
@@ -22,7 +22,7 @@ import (
 
 const (
 	authHeader     = "Authorization"
-	clusterParam   = "clusters"
+	clusterParam   = "cluster"
```

Comment:
This bug doesn't break the non-multicluster situation as below we default to the default cluster if the param is empty for backwards compat. at the moment.